CheckYourEligibility-Parent/wwwroot/js/cookieFunctions.js

```diff
@@ -20,28 +20,55 @@ function initializeClarity() {
 }
 
 function initCookieConsent() {
+    console.log("in innitCookieConsent");
     const hasChoice = cookie.read("cookie");
-    if (hasChoice==="true") {
+    if (hasChoice !== null) {
         initializeClarity();
-    }
-    
-    else if (hasChoice !== "false") {
+        document.getElementById('cookie-banner').style.display = 'none';
+        console.log("in innitCookieConsent - hasChoice not equal null");
```

Comment:
I'd remove these console.log, otherwise we'll flood user browsers with console logs.